### ISSUE TITLE
Cannot seem to use an exFAT card?

### ISSUE THREAD

**Issue by FatRakoon** [ZuluSCSI/ZuluSCSI-firmware#410]
Ok, first off, I am using an old ATARI TT, and the actual device is the AMIGAKIT version.

I had lots issues at first, but now I have it working perfectly fine.

However...

I cannot for the life of me get the thing to accept a card thats exFAT formatted. Its fine with FAT32 but not exFAT.
I have tried 13 cards now, 9 of them MicroSD in an adapter, and the other 4 are full sized, and all work in FAT32 but NONE work in exFAT.

It simply flashes the LED non stop.

What I am trying to do, is do a full copy of the TT Files that are on HARD DISKS, onto a card.

The TT is more than 30 years old and can only access up to 512MB Partitions, however I also use Mint and Magic and they access full FAT32 Partitions and under MINT I also have ext2fs. So for compatibility with the older stuff I have a few 512MB Partitions, but the FAT32 and the ext2fs partitions are both way over 4GB and so I really want to have exFAT to have bigger files on it.
Ideally, as the TT has a single HD inside it for its smaller Partitions, then a single 4GB for that drive would be great, but right now, I have only got up to 2GB partitions.

Hell... Im waffling about stuff you dont need to know.

Basically am I doing anything wrong?
Why cant I use an exFAT formatted SDCARD?
